docs/templating/twig_function_reference/storefront_twig_functions.md

```diff
@@ -0,0 +1,86 @@
+---
+description: Storefront Twig function enable controlling the rendering of storefront elements.
+edition: commerce
+---
+
+# Storefront Twig functions
```

Comment:
It would be worth to add 1-2 sentences of intro:
Examples of Twig functions that allow you to control the rendering of storefront elements.